docs/feature-store/feature-vectors.md

```diff
@@ -69,21 +69,35 @@ Due to the async nature of this action, the response object contains an `fv_resp
 
 `get_offline_features` expects to receive:
```

Comment:
I think you missing here few argument of this function : 

`:param with_indexes:    return vector with index columns and timestamp_key from the feature sets (default False)`
`:param update_stats:    update features statistics from the requested feature sets on the vector. Default is False.`
`:param engine:          processing engine kind ("local", "dask", or "spark")`
`:param engine_args:     kwargs for the processing engine`
`:param query:           The query string used to filter rows`
`:param spark_service:   Name of the spark service to be used (when using a remote-spark runtime)`